components/widgets/inputs/users_emails_input.jsx

```diff
@@ -311,18 +312,22 @@ export default class UsersEmailsInput extends React.PureComponent {
                 />
                 {this.props.showError && (
                     <div className='InputErrorBox'>
-                        <FormattedMarkdownMessage
-                            id={this.props.errorMessageId}
-                            defaultMessage={this.props.errorMessageDefault}
-                            values={this.props.errorMessageValues || null}
-                            disableLinks={true}
-                        >
-                            {(message) => (
-                                <components.NoOptionsMessage>
-                                    {message}
-                                </components.NoOptionsMessage>
-                            )}
-                        </FormattedMarkdownMessage>
+                        <div className='InputErrorBox__wrapper'>
+                            <FormattedMarkdownMessage
+                                id={this.props.errorMessageId}
+                                defaultMessage={this.props.errorMessageDefault}
+                                values={this.props.errorMessageValues || null}
+                                disableLinks={true}
+                            >
+                                {(message) => (
+                                    <components.NoOptionsMessage>
+                                        {message}
+                                    </components.NoOptionsMessage>
+                                )}
+                            </FormattedMarkdownMessage>
+                            {this.props.errorMessageId === 'invitation_modal.invite_members.hit_cloud_user_limit' &&
+                            <UpgradeLink/>}
```

Comment:
In order to keep this component generic and not have extra cloud logic here, could we pass in message as a component that the parent will set with the new `UpgradeLink` set as needed?